espnet2/bin/spk_inference.py

```diff
@@ -0,0 +1,551 @@
+        "If None, the 5 percent size of --max_cache_size",
+    )
+    group.add_argument("--shape_file", type=str, action="append", default=[])
+    group.add_argument(
+        "--input_size",
+        type=int_or_none,
+        default=None,
+        help="The number of input dimension of the feature",
+    )
+    group.add_argument("--allow_variable_data_keys", type=str2bool, default=False)
+    group.add_argument(
+        "--spk2utt",
+        type=str,
+        default="",
+        help="Directory of spk2utt file to be used in label mapping",
+    )
+    group.add_argument(
+        "--train_dtype",
+        default="float32",
+        choices=["float16", "float32", "float64"],
+        help="Data type for training.",
+    )
+    group.add_argument(
+        "--use_amp",
+        type=str2bool,
+        default=False,
+        help="Enable Automatic Mixed Precision. This feature requires pytorch>=1.6",
+    )
+    group.add_argument(
+        "--grad_clip",
+        type=float,
+        default=5.0,
+        help="Gradient norm threshold to clip",
+    )
+    group.add_argument(
+        "--accum_grad",
+        type=int,
+        default=1,
+        help="The number of gradient accumulation",
+    )
+    group.add_argument(
+        "--no_forward_run",
+        type=str2bool,
+        default=False,
+        help="Just only iterating data loading without "
+        "model forwarding and training",
+    )
+    group.add_argument(
+        "--grad_clip_type",
+        type=float,
+        default=2.0,
+        help="The type of the used p-norm for gradient clip. Can be inf",
+    )
+    group.add_argument(
+        "--grad_noise",
+        type=str2bool,
+        default=False,
+        help="The flag to switch to use noise injection to "
+        "gradients during training",
+    )
+    group.add_argument(
+        "--resume",
+        type=str2bool,
+        default=False,
+        help="Enable resuming if checkpoint is existing",
+    )
+    group.add_argument(
+        "--sort_in_batch",
+        type=str,
+        default="descending",
+        choices=["descending", "ascending"],
+        help="Sort the samples in each mini-batches by the sample "
+        'lengths. To enable this, "shape_file" must have the length information.',
+    )
+    group.add_argument(
+        "--sort_batch",
+        type=str,
+        default="descending",
+        choices=["descending", "ascending"],
+        help="Sort mini-batches by the sample lengths",
+    )
+    group.add_argument(
+        "--drop_last_iter",
+        type=str2bool,
+        default=False,
+        help="Exclude the minibatch with leftovers.",
+    )
+
+    group = parser.add_argument_group("The model configuration related")
+    group.add_argument(
+        "--spk_train_config",
+        type=str,
+        help="SPK training configuration",
+    )
+    group.add_argument(
+        "--spk_model_file",
+        type=str,
+        help="SPK model parameter file",
+    )
```

Comment:
Can you change the argument names to `--train_config` and `model_files` so that they are consistent with other tasks?